frontend/src/queries/useBlockQuery.ts

```diff
@@ -97,22 +77,15 @@ const BlockQueryByHash = gql<BlockByBlockHashResponse>`
 	}
 `
 
-type QueryParams = (
-	| {
-			id: Ref<string>
-			hash?: Ref<string>
-	  }
-	| {
-			hash: Ref<string>
-			id?: Ref<string>
-	  }
-) & {
+type QueryParams = {
+	hash?: Ref<string>
+} & {
```

Comment:
`hash` is no longer optional
```suggestion
	hash: Ref<string>
```